dynawo/sources/Simulation/DYNSimulation.cpp

```diff
@@ -753,12 +753,11 @@ void
 Simulation::simulate() {
   Timer timer("Simulation::simulate()");
 
-  string outputsDirectory = createAbsolutePath("outputs", context_->getInputDirectory());
-  if (!is_directory(outputsDirectory))
-    create_directory(outputsDirectory);
+  if (!is_directory(outputsDirectory_))
```

Comment:
This will break the integration in tools (CVG to not name it) that use Dynawo.

Reply:
Shit it is annoying to create this useless directory if we don't name the outputs directory this way...

Reply:
Yes but this is where the tool gets this file, and it is hard coded.

Reply:
ok